src/Camelotia.Presentation/ViewModels/CloudViewModel.cs

```diff
@@ -52,15 +53,24 @@ public sealed class CloudViewModel : ReactiveObject, ICloudViewModel, IActivatab
                 .WhenAnyValue(
                     x => x.Folder.IsVisible,
                     x => x.Rename.IsVisible,
-                    x => x.Auth.IsAuthenticated,
-                    (folder, rename, auth) => !folder && !rename && auth);
+                    (folder, rename) => !folder && !rename);
 
             _canInteract = canInteract
                 .ToProperty(this, x => x.CanInteract);
 
+            var canInteractAuthenticated = this
+                .WhenAnyValue(
+                    x => x.Folder.IsVisible,
+                    x => x.Rename.IsVisible,
+                    x => x.Auth.IsAuthenticated,
+                    (folder, rename, auth) => !folder && !rename && auth);
+
+            _canInteractAuthenticated = canInteractAuthenticated
+                .ToProperty(this, x => x.CanInteractAuthenticated);
+
             Refresh = ReactiveCommand.CreateFromTask(
                 () => cloud.GetFiles(CurrentPath),
-                canInteract);
+                canInteractAuthenticated);
```

Comment:
Thanks for the contribution! Probably worth removing an additional `ObservableAsPropertyHelper<T>` and declaring the `IObservable<bool>` locally, e.g. the following should just work:

```cs
var canRefresh = this
    .WhenAnyValue(
        x => x.CanInteract,
        x => x.Auth.IsAuthenticated,
        (canInteract, auth) => canInteract && auth);

 Refresh = ReactiveCommand.CreateFromTask(
    () => cloud.GetFiles(CurrentPath),
    canRefresh);
```

See also http://reactivex.io/documentation/operators/combinelatest.html